src/Playwright/Transport/Channels/APIRequestContextChannel.cs

```diff
@@ -62,13 +63,15 @@ public APIRequestContextChannel(string guid, Connection connection, APIRequestCo
             ["timeout"] = timeout,
             ["params"] = parameters?.ToProtocol(),
             ["headers"] = headers?.ToProtocol(),
-            ["jsonData"] = jsonData,
             ["postData"] = postData != null ? Convert.ToBase64String(postData) : null,
             ["formData"] = formData?.ToProtocol(),
             ["multipartData"] = multipartData?.ToProtocol(),
-        };
-
-        var response = await Connection.SendMessageToServerAsync(Guid, "fetch", message).ConfigureAwait(false);
+        }.FilterNullValues();
```

Comment:
Isn't the null value filtering already done here?
https://github.com/microsoft/playwright-dotnet/blob/ffb7c21272c3f078c95976e7df60a41d9c53fbb7/src/Playwright/Transport/Connection.cs#L133-L138

Reply:
Ah you removed it, should have scrolled to the end before commenting 😆